src/status_im/signals/core.cljs

```diff
@@ -60,27 +60,29 @@
   [{:keys [db] :as cofx} event]
   (log/info "local pairing signal received"
             {:event event})
-  (let [connection-success?             (= (:type event)
-                                           constants/local-pairing-event-connection-success)
+  (let [connection-success?             (and (= (:type event)
```

Comment:
i thought `role` would be used within this function 🙂

Reply:
Hi @qfrank! 

We haven't used the `role` yet because it will be needed (and used) for this issue https://github.com/status-im/status-mobile/issues/15356 to take the sender and receiver to the appropriate screens upon completion.

Apologies, I forgot to add this note to the PR description.